doc/API/centreon-api-v2.yaml

```diff
@@ -1646,7 +1676,7 @@ components:
                 url:
                   type: string
                   description: "Url of the icon"
-                  example: "/media/linux.png"
+                  example: "./img/media/topology/firewall.png"
```

Comment:
Is the path going to be relative?

Reply:
yes. Is it better to manipulate an absolute path on frontend ?

Reply:
final decision: use absolute path and frontend will add root path (eg: /centreon)